themes/bootstrap3/js/common.js

```diff
@@ -706,12 +706,15 @@ function unwrapJQuery(node) {
 }
 
 function setupJumpMenus(_container) {
-  var container = _container || $('body');
-  container.find('select.jumpMenu').on("change", function jumpMenu() {
-    // Check if jumpMenu is still enabled (search.js may have disabled it):
-    if ($(this).hasClass('jumpMenu')) {
-      $(this).parent('form').trigger("submit");
-    }
+  var container = _container || document.body;
```

Comment:
_container may be a jQuery object, based on usage in record.js. Should we add an unwrapJQuery() call somewhere to ensure things are consistent?